src/UI/Components/UpdateBanner/UpdateBanner.tsx

```diff
@@ -1,27 +1,26 @@
-import React, { useContext, useState } from "react";
+import React, { useContext, useEffect, useState } from "react";
 import { Banner, Flex } from "@patternfly/react-core";
-import { ApiHelper } from "@/Core";
-import { GetVersionFileQueryManager } from "@/Data/Managers/GetVersionFile/OnteTimeQueryManager";
+import { useGetVersionFileInfo } from "@/Data/Managers/V2/Miscellaneous";
 import { DependencyContext } from "@/UI/Dependency";
 import { words } from "@/UI/words";
 
-interface Props {
-  apiHelper: ApiHelper;
-}
-
-export const UpdateBanner: React.FunctionComponent<Props> = (props) => {
+/**
+ * Component for displaying an update banner when a new version is available.
```

Comment:
would be nice to be complete here and specify this is about the app version. For someone new to this codebase, it might quickly become confusing with all the different version-terms available.